specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/WebApps.json

```diff
@@ -262,6 +262,15 @@
           "204": {
             "description": "Successfully deleted web app."
           },
+          "202": {
+            "description": "Asynchronous operation in progress.",
+            "headers": {
+              "location": {
```

Comment:
ARM's guidance is here:
https://armwiki.azurewebsites.net/rpaas/async.html#creating-or-updating-resources-put
Put/Patch/Post

Seems like the header to use is `Azure-AsyncOperation`. Should we use that? 

I noticed that the examples all use Azure-AsyncOperation for some reason